server/index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const routes = require('./routes');
+const app = express();
+const port = 3001;
```

Comment:
Some modern hosting services (like Heroku) don't deploy from static ports, so you'll need to account for that here. Basically, if the host is using a port already, use that, if not, then you can use 3001. Code would be `const port = process.env.PORT || 3001`

Reply:
made that change thanks